rerun_py/rerun_sdk/rerun/_rerun2/datatypes/rotation_axis_angle_ext.py

```diff
@@ -16,3 +18,21 @@ def angle__field_converter_override(x: AngleLike) -> Angle:
             return x
         else:
             return Angle(rad=x)
+
+    @staticmethod
+    def native_to_pa_array_override(data: RotationAxisAngleArrayLike, data_type: pa.DataType) -> pa.Array:
+        from . import AngleArray, RotationAxisAngle, Vec3DArray
+
+        if isinstance(data, RotationAxisAngle):
+            data = [data]
+
+        axis_pa_array = Vec3DArray._native_to_pa_array([rotation.axis for rotation in data], data_type["axis"].type)
```

Comment:
I assume we use `_native_to_pa_array` here instead of `from_similar` to avoid the `wrap_array` extension wrapper?